<launch>
  <rosparam command="load" file="parameter_.yaml" />
  <!--group ns="group1"-->
    <!--node name="node_ask_cost" pkg="ros_wifi" type="service_ask_cost.py" /-->
    <!--node name="node_ask_node" pkg="ros_wifi" type="service_ask_node.py" /-->
    <!--node name="node_ask_data" pkg="ros_wifi" type="service_ask_data.py" /-->
    <!--node name="node_wifi_conf" pkg="ros_wifi" type="service_wifi_confirm.py" /-->
  <node name="node_client" pkg="ros_wifi" type="wifi_client_send.py" output="screen"/>
  <node name="node_master" pkg="ros_wifi" type="wifi_master_process.py" output="screen"/>
  <node name="node_server" pkg="ros_wifi" type="wifi_server_receive.py" output="screen"/>
  <!--/group-->
  
</launch>
<!--10.0.2.255 cpu, 192.168.1.101 ng, 192.168.1.102 danlu
-->
